data/meltano.yml

```
@@ -2,10 +2,16 @@ version: 1
 default_environment: userdev
 send_anonymous_usage_stats: true
 project_id: c15e971a-d318-4a9d-979b-1039ce5fd1b1
+
 include_paths:
 - ./environments/*.meltano.yml
 - ./extract/*.meltano.yml
 - ./load/*.meltano.yml
 - ./orchestrate/*.meltano.yml
 - ./transform/*.meltano.yml
 - ./utilities/*.meltano.yml
+
+# Feature Flags - https://docs.meltano.com/reference/settings#feature-flags
+
+# fail if any referenced env vars are missing
+ff.env_var_strict_mode: true
```

Per https://github.com/meltano/meltano/pull/6680

```suggestion
ff.strict_env_var_mode: true
```